.github/workflows/rpm-build-and-test-report.yml

```diff
@@ -41,7 +41,7 @@
             esac
             echo "STAGE_NAME=Build RPM on $DISTRO_NAME $DISTRO_VERSION" >> $GITHUB_ENV
       - name: Test Report
-        uses: phoenix-actions/test-reporting@v10
+        uses: phoenix-actions/test-reporting@v15
```

Comment:
Indeed.  It would seem to me that if we are going to rely on Dependabot to update these as necessary, then we may as well keep the Scorecard scanning happy and pin them.